samples/react/.eslintrc

```diff
@@ -54,6 +55,30 @@
     "react/no-array-index-key": "off",
     "no-use-before-define": 0,
     "global-require": 0,
-    "no-param-reassign": 0
+    "no-param-reassign": 0,
+    "no-useless-escape": "off",
+    "no-restricted-globals": "off",
```

Comment:
Some relevant comments made on the base .eslintrc file:

1. Should be able to remove the following (defaults):
```
"no-restricted-globals": "off",
"no-restricted-properties": "off",
"max-len": "off",
"no-debugger": "error",
"no-empty": "error",
"no-fallthrough": "error",
```

2. "no-console" rule doing much with all of the allows?

Reply:
Removed, removed no-console